From 8283877390fcf6c6682921d5c9f4743a0fc41112 Mon Sep 17 00:00:00 2001
From: Mitch Harder <mitch.harder@sabayonlinux.org>
Date: Sun, 13 May 2012 12:16:26 -0500
Subject: Modify Makefile to allow optional ACL dependency.

Modify the Makefile to make the 'btrfs-convert' build optional
since this is the only program depending on ACL.
---
 Makefile |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 79818e6..3b25220 100644
--- a/Makefile
+++ b/Makefile
@@ -17,11 +17,11 @@ prefix ?= /usr/local
 bindir = $(prefix)/bin
 LIBS=-luuid
 RESTORE_LIBS=-lz
 
 progs = btrfsctl mkfs.btrfs btrfs-debug-tree btrfs-show btrfs-vol btrfsck \
-	btrfs btrfs-map-logical btrfs-image btrfs-zero-log btrfs-convert \
+	btrfs btrfs-map-logical btrfs-image btrfs-zero-log \
 	btrfs-find-root btrfs-restore btrfstune
 
 # make C=1 to enable sparse
 ifdef C
 	check = sparse $(CHECKFLAGS)
@@ -112,7 +112,10 @@ clean :
 	cd man; make clean
 
 install: $(progs) install-man
 	$(INSTALL) -m755 -d $(DESTDIR)$(bindir)
 	$(INSTALL) $(progs) $(DESTDIR)$(bindir)
+	if [ -e btrfs-convert ]; \
+		then $(INSTALL) btrfs-convert $(DESTDIR)$(bindir); \
+	fi
 
 -include .*.d
-- 
1.7.3.4

